tests/test_octodns_manager.py

```diff
@@ -191,6 +191,11 @@ def test_aliases(self):
                               'zone alias.tests. is an alias zone',
                               text_type(ctx.exception))
 
+            # Sync an alias without the zone it refers to
+            tc = Manager(get_config_filename('simple-alias-zone.yaml')) \
+                .sync(eligible_zones=["alias.tests."])
+            self.assertEquals(0, tc)
```

Comment:
Looks good. As mentioned in https://github.com/github/octodns/issues/633#issuecomment-735903849, I think the desired behavior here would be to throw a `ManagerException` with an error message about syncing an aliased zone without it's companion.